src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -115,6 +115,11 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
     }
     telemetry.add_extension_event('connectedk8s', kubernetes_properties)
 
+    # Checking if it is an AKS cluster
+    is_aks_cluster = check_aks_cluster(kube_config, kube_context)
```

Comment:
I think we should not raise errors anywhere inside this function. Just log warnings in case of exceptions and move ahead. Also no need for exception telemetry/setting user faults. If there is a problem with the kubeconfig, it will be flagged when we load the kube config during onboarding(with appropriate telemetry).

Reply:
If any kubeconfig issue is there, anyway that will be raised as clierror either in **check_kube_connection** or **config.load_kube_config(config_file=kube_config, context=kube_context)**. And i am doing this aks check afterwards.

Reply:
Yeah okay, will raise warnings.

Reply:
Yes, failing here anyways wont make sense since we are only checking if the cluster is aks or not and the validation has already been done when loading kube_config